resources/quality/ultimaker_s3/um_s3_cc0.6_um-pla_0.2mm.inst.cfg

```
@@ -0,0 +1,29 @@
+[general]
+definition = ultimaker_s3
+name = Fast - Experimental
+version = 4
+
+[metadata]
+is_experimental = True
+material = ultimaker_pla
+quality_type = draft
+setting_version = 22
+type = quality
+variant = CC 0.6
+weight = -2
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```